vespalib/src/vespa/vespalib/net/tls/transport_security_options.cpp

```diff
@@ -0,0 +1,12 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+
+#include "transport_security_options.h"
+#include <openssl/crypto.h>
+
+namespace vespalib::net::tls {
+
+TransportSecurityOptions::~TransportSecurityOptions() {
+    OPENSSL_cleanse(_private_key_pem.data(), _private_key_pem.size());
```

Comment:
minor layer violation

Reply:
Agreed, but will leave this for now since it only calls OpenSSL and not into the `impl/` submodule. Secure erasing functionality is a likely candidate to be moved to a separate crypto utility library in vespalib.